jetty-server/src/main/java/org/eclipse/jetty/server/LowResourceMonitor.java

```diff
@@ -95,14 +88,79 @@ public void run()
             if (isRunning())
             {
                 monitor();
-                _scheduler.schedule(_monitor,_period,TimeUnit.MILLISECONDS);
+                _scheduler.schedule( _monitor, _period, TimeUnit.MILLISECONDS);
             }
         }
     };
 
+    private boolean _monitorThreads=true;
+    private int _maxConnections;
```

Comment:
Just for containment... I'd probably move things like `_maxConnections` to be fields of the checker class that uses them.   Only keep fields here if they are used by more than one check.  Eventually users should be encouraged to configure these by calling setters on the checkers rather than setters here (which can eventually be deprecated)

Reply:
agree but my idea was to keep backward compat but yes can be a deprecated with an extra layer